metadata_service/proxy/neo4j_proxy.py

```diff
@@ -592,7 +592,8 @@ def add_badge(self, *,
                   id: str,
                   badge_name: str,
                   category: str = '',
-                  resource_type: ResourceType = ResourceType.Table) -> None:
+                  resource_type: ResourceType,
+                  column_name: str = '') -> None:
```

Comment:
I'm guessing I'll need to change the query below but not sure how to approach that.